README.md

```diff
@@ -1,35 +1,89 @@
 ![](./resources/official_armmbed_example_badge.png)
-# Example cellular application for Mbed OS
+# Cellular Example
 
 This is an example based on `mbed-os` cellular APIs that demonstrates a TCP or UDP echo transaction with a public echo server.
 
-(Note: To see this example in a rendered form you can import into the Arm Mbed Online Compiler, please see [the documentation](https://os.mbed.com/docs/mbed-os/latest/apis/cellular-api.html#cellular-example-connection-establishment).)
+(Note: To see this example in a rendered form you can import into the Arm Mbed Online Compiler, please see
+[the documentation](https://os.mbed.com/docs/mbed-os/latest/apis/cellular-api.html#cellular-example-connection-establishment).)
 
 ## Getting started
 
 This particular cellular application uses a cellular network and network-socket APIs that are part of [`mbed-os`](https://github.com/ARMmbed/mbed-os).
 
-The program uses a [cellular modem driver](https://github.com/ARMmbed/mbed-os/tree/master/features/cellular/framework/API) using an external IP stack (LWIP) standard 3GPP AT 27.007 AT commands to setup the cellular modem and registers to the network.
+The program uses a [cellular modem driver](https://github.com/ARMmbed/mbed-os/tree/master/connectivity/cellular/include/cellular/framework/API)
+using an external IP stack (LWIP) standard 3GPP AT 27.007 AT commands to setup the cellular modem and registers to the network.
 
-After registration, the driver opens a point-to-point protocol (PPP) pipe using LWIP with the cellular modem and connects to internet. This driver currently supports UART data connection type only between your cellular modem and MCU.
+After registration, the driver opens a point-to-point protocol (PPP) pipe using LWIP with the cellular modem and connects
+to internet. This driver currently supports UART data connection type only between your cellular modem and MCU.
 
-For more information on Arm Mbed OS cellular APIs and porting guide, please visit the [Mbed OS cellular API](https://os.mbed.com/docs/latest/reference/cellular.html) and [contributing documentation](https://os.mbed.com/docs/mbed-os/latest/contributing/index.html).
+For more information on Arm Mbed OS cellular APIs and porting guide, please visit the
+[Mbed OS cellular API](https://os.mbed.com/docs/mbed-os/latest/apis/cellular-networking.html) and
+[Mbed OS cellular porting guide](https://os.mbed.com/docs/mbed-os/latest/porting/cellular-device-porting.html).
 
-### Download the application
+### Board support
+
+The cellular modem driver in this example uses PPP with an Mbed-supported external LwIP stack.
+It supports targets when modem exists on the Mbed Enabled target as opposed to plug-in modules (shields).
```

Comment:
```suggestion
The driver supports Mbed Enabled targets when a modem exists on the device. Plug-in modules (shields) are not supported.
```

It wasn't clear to me whether it supported shields or not from the way this sentence was written. My suggestion assumes we don't support shields.